.github/workflows/mayhem.yml

```diff
@@ -0,0 +1,60 @@
+name: Mayhem
```

Comment:
Can you update the versions for these actions to the latest published version? For example I know upload-sarif is at v2 now.